lib/validator/v3/validator.go

```diff
@@ -233,6 +234,11 @@ func validatePortName(v *validator.Validate, topStruct reflect.Value, currentStr
 	return len(s) != 0 && len(k8svalidation.IsValidPortName(s)) == 0
 }
 
+func validateMustBeNil(v *validator.Validate, topStruct reflect.Value, currentStructOrField reflect.Value, field reflect.Value, fieldType reflect.Type, fieldKind reflect.Kind, param string) bool {
+	log.Debugf("Validate v1 IPIP. Must be nil: %s", field.String())
```

Comment:
This is written as a generic validator, so probably should not include the field name in the log unless you are getting it from the parameters.

Reply:
hmm yeah I was also feeling effy about that